packages/frontend/app/styles/tailwind.css

```diff
@@ -1,3 +1,8 @@
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
+
+a.tooltip {
+  color: revert;
+  text-decoration: revert;
+}
```

Comment:
My understanding is this restores some original styling for `a` links that tailwind strips out.

This seems like a good way to do it. We've handled this differently elsewhere (a bunch of tailwind utility classnames) but I think what you've done here is simpler and easier to use. 

If it makes sense to you, please merge this PR which standardizes on this method: https://github.com/interledger/rafiki/pull/2719. I also renamed the class to `default-link` and updated reference since it would apply to more than tooltips.

Reply:
thanks for the review, merged in